haproxy/datadog_checks/haproxy/haproxy.py

```diff
@@ -704,6 +758,26 @@ def _process_metrics(
                 except ValueError:
                     pass
 
+    def _process_stick_table_metrics(
+        self, data, services_incl_filter=None, services_excl_filter=None, custom_tags=None
+    ):
+        """
+        Stick table metrics processing. Two metrics will be created for each stick table (current and max size)
+        """
+
+        custom_tags = [] if not custom_tags else custom_tags
+
+        for line in data:
+            table = StickTable.parse(line)
+            if table is None:
+                continue
+            if self._is_service_excl_filtered(table.name, services_incl_filter, services_excl_filter):
+                continue
+
+            tags = ["haproxy_service:%s" % table.name, "type:%s" % table.type] + custom_tags
```

Comment:
is this ok or should it be somewhat more specific, like `stick_table_type` ?

Reply:
Yeah or `stick_type`?